docs/features/vendored-dependencies.md

```diff
@@ -71,3 +73,27 @@ We also support json-formatted dependencies:
   ]
 }
 ```
+## How Vendored Dependencies are scanned
+
+There are two methods of vendored dependency scanning: "CLI license scan" and "archive upload".
+
+The default is typically "CLI license scan", but your organization may have opted to default to "archive upload".
+
+Both methods use the same license scanning technology and will give you the same results. The difference is where the license scan is done.
+
+A "CLI license scan" runs a license scan directly on the server where you are running `fossa analyze`, and only uploads the matched license data to FOSSA's servers.
+
+"Archive upload" uploads the files at the specified path to a secure S3 bucket. We license scan the uploaded files on our servers. All files that do not contain licenses are then removed after 30 days.
```

Comment:
This used to say "after 2 weeks", but I was just looking at our S3 lifecycle configuration for something else and I'm pretty sure that this should be 30 days.